includes/validation/class-amp-validation-manager.php

```diff
@@ -699,7 +699,9 @@ public static function filter_the_editor_to_detect_sources( $output ) {
 		$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace -- Only way to find theme/plugin responsible for calling.
 		foreach ( $backtrace as $call ) {
 			if (
-				isset( $call['function'], $call['file'] )
+				! empty( $call['function'] )
+				&&
+				! empty( $call['file'] )
```

Comment:
Apparently `isset($call['file'])` is fine since the `file` is nullable, but `function` must be accessed via `! empty($call['function'])` since it is not nullable. The `function` check is probably overkill in this case, but it doesn't hurt.